Dockerfile

```diff
@@ -15,7 +15,7 @@
 FROM debian:stretch
 MAINTAINER Stefan Schimanski <sttts@redhat.com>
 RUN apt-get update \
- && apt-get install -y -qq git=1:2.11.0-3+deb9u4 \
+ && apt-get install -y -qq git=1:2.11.0-3+deb9u5 \
```

Comment:
?

Reply:
Debian updated git and old one is gone. We have this explicit version here to notice when git changes.